ietf-layer0-types-ext.yang

```diff
@@ -568,6 +568,31 @@ module ietf-layer0-types-ext {
     }
   }  
 
+  grouping pdl-penalty {

```

Comment:
This grouping is used only once which usually triggers some comments from YANG doctors.

Unless there is an intention to re-use this grouping somewhere else, these two attributes can be place directly in the code below.

Reply:
To be addressed in issue #24